apps/wrapper/src/renderer/src/main.ts

```diff
@@ -2,7 +2,7 @@ import './assets/tailwind.css';
 import App from './App.svelte';
 
 const app = new App({
-  target: document.getElementById('app'),
+  target: document.querySelector('#app') as Element,
```

Comment:
I would prefer using the `!` syntax as opposed to overwriting the type. But I think this works fine as long as the compiler does not complain

E.g.,
```
target: document.querySelector('#app')!,
```

This tells Typescript that you are sure that there is such an element in the `index.html` file.